pkg/apis/backups/v1alpha1/common_type.go

```diff
@@ -34,13 +34,6 @@ type Destination struct {
 	Clusters []*corev1.ObjectReference `json:"clusters,omitempty"`
 }
 
-type ClusterSelector struct {
```

Comment:
why delete clusterSelector?
Are you sure the backup functions properly after deletion?

Reply:
type `ClusterSelector` is not referenced anywhere